src/redis-benchmark.c

```diff
@@ -413,6 +416,15 @@ static void resetClient(client c) {
     c->pending = config.pipeline;
 }
 
+static long calculateClientDuration(long long connection_duration_usec, long max_jitter_usec){
+    long jitter = 0;
+    if (config.connection_duration_jitter_usec > 0){
```

Comment:
not sure why we pass the arguments here and still check the config inside. I suggest maybe we can just query the config values inside the function body?

Reply:
Fixed